<!DOCTYPE html>
<!--
                                                                          
                                 .--,-``-.                              
                                /   /     '.                            
                               / ../        ;                           
      ,---,                    \ ``\  .`-    '                   ,---,  
  ,-+-. /  |               .---.\___\/   \   :               ,-+-. /  | 
 ,--.'|'   |  ,--.--.    /.  ./|     \   :   |      ,---.   ,--.'|'   | 
|   |  ,"' | /       \ .-' . ' |     /  /   /      /     \ |   |  ,"' | 
|   | /  | |.--.  .-. /___/ \: |     \  \   \     /    / ' |   | /  | | 
|   | |  | | \__\/: . .   \  ' . ___ /   :   |   .    ' /  |   | |  | | 
|   | |  |/  ," .--.; |\   \   '/   /\   /   :   '   ; :__ |   | |  |/  
|   | |--'  /  /  ,.  | \   \  / ,,/  ',-    .___'   | '.'||   | |--'   
|   |/     ;  :   .'   \ \   \ \ ''\        ;/  .\   :    :|   |/       
'---'      |  ,     .-./  '---" \   \     .' \  ; \   \  / '---'        
            `--`---'             `--`-,,-'    `--" `----'               
                      
                     2018/4/26   www.nav3.cn

        ,+++++++++++++++++++++++++++++++++++++++++++++++++.
        ( We want you! https://github.com/xjh22222228/nav )
        `-,+++++++++++++++++++++++++++++++++++++++++++++++'
-->
<html data-critters-container>
  <head>
    <meta charset="utf-8">
    <base href="./">
    <meta name="author" content="https://github.com/xjh22222228">
    <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <meta name="renderer" content="webkit">
    <!-- nav.config-start --><!-- https://github.com/xjh22222228/nav -->
  <title>Bioinfo Explorer</title>
  <meta property="og:title" content="Bioinfo Explorer">
  <meta property="og:description" content="Curated Bioinformatics Resources">
  <meta property="og:type" content="website">
  <meta name="description" content="Curated Bioinformatics Resources">
  <meta name="keywords" content="Bioinfomatics, Navigation, Programming" id="xjh_2">
  <link rel="icon" href="https://gcore.jsdelivr.net/gh/GilbertHan1011/nav@image/nav-1731223335526-nav_log.png">
  <link rel="apple-touch-icon" href="https://gcore.jsdelivr.net/gh/GilbertHan1011/nav@image/nav-1731223335526-nav_log.png">
  <link rel="prefetch" href="//unpkg.com/ng-zorro-antd@18.1.1/ng-zorro-antd.dark.min.css"><!-- nav.config-end -->
    <link rel="stylesheet" href="assets/fonts/iconfont.css" media="print" onload="this.media='all'"><noscript><link rel="stylesheet" href="assets/fonts/iconfont.css"></noscript>
    <!-- nav.headerContent-start -->
    <!-- nav.headerContent-end -->
  <style>*,:before,:after{--tw-border-spacing-x:0;--tw-border-spacing-y:0;--tw-translate-x:0;--tw-translate-y:0;--tw-rotate:0;--tw-skew-x:0;--tw-skew-y:0;--tw-scale-x:1;--tw-scale-y:1;--tw-pan-x: ;--tw-pan-y: ;--tw-pinch-zoom: ;--tw-scroll-snap-strictness:proximity;--tw-gradient-from-position: ;--tw-gradient-via-position: ;--tw-gradient-to-position: ;--tw-ordinal: ;--tw-slashed-zero: ;--tw-numeric-figure: ;--tw-numeric-spacing: ;--tw-numeric-fraction: ;--tw-ring-inset: ;--tw-ring-offset-width:0px;--tw-ring-offset-color:#fff;--tw-ring-color:rgb(59 130 246 / .5);--tw-ring-offset-shadow:0 0 #0000;--tw-ring-shadow:0 0 #0000;--tw-shadow:0 0 #0000;--tw-shadow-colored:0 0 #0000;--tw-blur: ;--tw-brightness: ;--tw-contrast: ;--tw-grayscale: ;--tw-hue-rotate: ;--tw-invert: ;--tw-saturate: ;--tw-sepia: ;--tw-drop-shadow: ;--tw-backdrop-blur: ;--tw-backdrop-brightness: ;--tw-backdrop-contrast: ;--tw-backdrop-grayscale: ;--tw-backdrop-hue-rotate: ;--tw-backdrop-invert: ;--tw-backdrop-opacity: ;--tw-backdrop-saturate: ;--tw-backdrop-sepia: ;--tw-contain-size: ;--tw-contain-layout: ;--tw-contain-paint: ;--tw-contain-style: }*,:before,:after{box-sizing:border-box;border-width:0;border-style:solid;border-color:#e5e7eb}:before,:after{--tw-content:""}html{line-height:1.5;-webkit-text-size-adjust:100%;tab-size:4;font-family:ui-sans-serif,system-ui,sans-serif,"Apple Color Emoji","Segoe UI Emoji",Segoe UI Symbol,"Noto Color Emoji";font-feature-settings:normal;font-variation-settings:normal;-webkit-tap-highlight-color:transparent}body{margin:0;line-height:inherit}p{margin:0}html,body{width:100%;height:100%}*,*:before,*:after{box-sizing:border-box}html{font-family:sans-serif;line-height:1.15;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%;-ms-overflow-style:scrollbar;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{margin:0;color:#000000d9;font-size:14px;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,"Apple Color Emoji","Segoe UI Emoji",Segoe UI Symbol,"Noto Color Emoji";font-variant:tabular-nums;line-height:1.5715;background-color:#fff;font-feature-settings:"tnum"}p{margin-top:0;margin-bottom:1em}html{--antd-wave-shadow-color:#1890ff;--scroll-bar:0}@charset "UTF-8";:root{--componentHeight:155px}html{-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}body{min-height:100vh;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;-webkit-overflow-scrolling:touch;font-size:14px}</style><link rel="stylesheet" href="styles.0d842822ce1c1af9.css" media="print" onload="this.media='all'"><noscript><link rel="stylesheet" href="styles.0d842822ce1c1af9.css"></noscript></head>
  <script>
    window.__TITLE__ = document.title
  </script>
  <body class="dark-scrollbar-border" id="x-j-h">
    <app-xiejiahe>
      <!-- nav.loading-start --><style>.l__1{z-index:999;position:fixed;top:0;left:0;right:0;bottom:0;background-color:#fff;display:flex;align-items:center;justify-content:center;flex-direction:column;}.l__2{width:200px;height:200px;background:url("https://gcore.jsdelivr.net/gh/xjh22222228/public@gh-pages/nav/0020f4d5-9f70-4b7f-87d4-61310d1db8d1.svg");background-repeat:repeat;}.l__3{font-size:18px;margin-top:15px;font-weight:bold;color:#000;}</style>
<div class="l__1"><div class="l__2"></div><div class="l__3">......</div></div><!-- nav.loading-end -->
    </app-xiejiahe>

    <!-- nav.seo-start -->
<div data-url="https://github.com/xjh22222228/nav" data-server-time="1731843660557" data-a="x.i.e-jiahe" data-date="2024-11-17 19:41:00" id="META-NAV" style="z-index:-1;position:fixed;top:-10000vh;left:-10000vh;">
<div>Bioinformatics Algorithms</div><p>Bioinformatics algorithms book</p><div>ROSALIND</div><p>ROSALIND programming website</p><div>Computational Biology - Genomes, Networks, and Evolution</div><p>This text covers the algorithmic and machine learning foundations of computational biology combining theory with practice. We cover both foundational topics in computational biology, and current research frontiers. We study fundamental techniques, recent advances in the field, and work directly with current large-scale biological datasets.</p><div>MLCB24 - Machine Learning in Cell Biology 2024</div><div>Training Courses of Babraham Institute</div><p>As part of its work with the Babraham Institute, the Bioinformatics group runs a regular series of training courses on many aspects of bioinformatics.

These courses are run regularly on the Babraham site but we are also able to come out and present them on other sites and also deliver them remotely. You can see the list of current Babraham dates which are available, and you can contact us to discuss options for running courses on your site.</p><div>Comp790-166: Computational Biology</div><p>Modern, high-throughput assays allow us to efficiently profile a variety of biological processes to gain a systems-level understanding of health and disease. Recent technologies and experimental assays generate an abundance of detailed information that needs to be extracted, summarized, and interpreted. In this course we will discuss the methodology used to extract signal from (e.g. process, engineer features from, combine, etc.) data generated by some of the most cutting-edge experimental paradigms, such as single-cell assays and imaging. We will go into detail about the methods and theory underlying bioinformatics algorithms, originating from numerical linear algebra, graph-signal processing, and machine learning. While computational biology is a very broad field, we will focus here on applications in single-cell biology (CyTOF, single-cell RNA sequencing), multiomics/multi-modal analysis, systems immunology, and benchmarking. For each class of algorithms introduced for some task on biological data, we will also go over necessary theory and mathematical intuition. The course covers the foundations for biomedical data science and does not assume any biological knowledge.</p><div>Bioinformatics Tutorial By Lu Zhi</div><p>Bioinformatics Tutorial By Lu Zhi Lab</p><div>Computational Genomics with R</div><p>The aim of this book is to provide the fundamentals for data analysis for genomics. We developed this book based on the computational genomics courses we are giving every year.</p><div>Modern Statistics for Modern Biology</div><p>Modern Statistics for Modern Biology, by Susan Holmes and Wolfgang Huber</p><div>Misc notes</div><p>Various notes collected over the years</p><div>The Biostar Handbook: 2nd Edition</div><p>A guide to bioinformatics data analysis</p><div>GitHub - crazyhottommy/getting-started-with-genomics-tools-and-resources: Unix, R and python tools for genomics and data science</div><p>Unix, R and python tools for genomics and data science - crazyhottommy/getting-started-with-genomics-tools-and-resources</p><div>harvardx</div><p>HarvardX Biomedical Data Science By rafalab</p><div>Introduction to Bioinformatics and Computational Biology</div><p>Introduction to Bioinformatics and Computational Biology By Xiaole Liu Lab</p><div>Orchestrating Single-Cell Analysis with Bioconductor</div><p>Bioconductor's scRNA tutorial</p><div>Single-cell best practices</div><div>GWAS ATLAS</div><p>This atlas is a database of publicly available GWAS summary statistics. Each GWAS can be browsed with the manhattan plot, risk loci, MAGMA (i.e. gene-based) results, SNP heritability and genetic correlations with other GWAS in the database. 600 GWAS were performed in this project based on UK Biobank release 2 data under application ID 16406. Full summary statistics can be downloaded from the original source following the provided links.</p><div>ChIP-Atlas</div><p>A data-mining suite for exploring epigenomic landscapes by fully integrating 428,000 ChIP-seq, ATAC-seq and Bisulfite-seq experiments.</p><div>ARCH4</div><p>All RNA-seq and ChIP-seq sample and signature search (ARCHS4) (https://maayanlab.cloud/archs4/) is a resource that provides access to gene and transcript counts uniformly processed from all human and mouse RNA-seq experiments from the Gene Expression Omnibus (GEO) and the Sequence Read Archive (SRA).</p><div>refine.bio</div><p>refine.bio Compendia packages the data processed by refine.bio pipelines for flexible and broad use by computational biologists and data scientists</p><div>Google docs: Single cell study database</div><div>Single-cell Pan-species atlas</div><p>SPEED is the largest pan-species single cell database at present, covering 127 species to date. SPEED also collects various scRNA-seq atlases in evolution, development and disease, providing an ecological and evolutionary perspective for the study of development and disease.</p><div>Single Cell Portal</div><p>Single Cell Portal by BroadInstitute</p><div>CELLxGENE</div><p>scRNA datasets collections</p><div>Principles of Spatial Transcriptomics Analysis with Bioconductor</div><p>This is the website for the online book Principles of Spatial Transcriptomics Analysis with Bioconductor.

This book provides interactive examples and discussion on key principles of computational analysis workflows for spatial transcriptomics data using Bioconductor in R. The book contains chapters describing individual analysis steps as well as extended workflows, each with examples including R code and datasets.

The book is organized into several parts, consisting of introductory materials, and analysis steps and workflows for sequencing-based and imaging-based spatial transcriptomics platforms.</p><div>Zhonglab's 3D genome tutorial</div><p>Zhonglab's 3D genome tutorial</p><div>Ming Tang's ChIP-seq notes</div><p>Ming Tang's ChIP-seq notes</p><div>Hi-C Data Analysis Bootcamp</div><p>Hi-C Data Analysis Bootcamp</p><div>Github: Hi-C data analysis tools and papers</div><p>Hi-C data analysis tools and papers collection</p><div>Orchestrating Hi-C analysis with Bioconductor</div><p>Orchestrating Hi-C analysis with Bioconductor</p><div>Cooltools  documentation</div><p>Cooltools tutorial</p><div>Distill</div><p>Machine Learning Research
Should Be Clear, Dynamic and Vivid.
Distill Is Here to Help.</p><div>Nature Review Genetics Collection : Machine learning in genomics</div><p>Machine learning has made significant contributions to the field of genetics, revolutionizing the way researchers analyse and interpret the vast amounts of genomic data. This collection brings together Reviews written by key opinion leaders in the field that explain cutting-edge machine learning methodology and how these tools are being applied in specific research areas. From predictive modelling to pattern recognition, this collection showcases the innovative ways in which machine learning is offering unprecedented insights into genetic variation, disease mechanisms and evolutionary dynamics.</p><div>Deep Learning in Genomics</div><p>Can deep learning models that have defeated gamers or recognized images better than humans also help us understand genomics? How far will this interdisciplinary research take us on our quest to cure cancer? In an era with faster-than-Moore’s-Law exponential growth of the genomics data (Berger et al. 2016), deep learning methods are finally able to assist in solving essential problems in the field. However, these exciting developments also face challenges that are unique to working with data from our DNA.

As researchers trying to combine deep learning and genomics, we have to think carefully about applying these models effectively to genomics tasks. Is it appropriate to use deep learning for our application? What model should we use? Will our approach improve our understanding of the data or the problem? In this course, you will answer these questions by our coverage of recent research literature in the class. You will learn about different genomics tasks, deep learning models, and how they fit together. The course is designed to enable critical thinking and allows students to work together to apply these models.</p><div>Deep Learning in the Life Sciences</div><p>This courses introduces foundations and state-of-the-art machine learning challenges in genomics and the life sciences more broadly. We introduce both deep learning and classical machine learning approaches to key problems, comparing and contrasting their power and limitations. We seek to enable students to evaluate a wide variety of solutions to key problems we face in this rapidly developing field, and to execute on new enabling solutions that can have large impact. As part of the subject students will implement solutions to challenging problems, first in problem sets that span a carefully chosen set of tasks, and then in an independent project. Students will program using Python 3 and TensorFlow 2 in Jupyter Notebooks, a nod to the importance of carefully documenting your work so it can be precisely reproduced by others.</p><div>Github: Awesome statistics</div><p>The repository consists of a dataset with curated links to material dealing with statistics and data. There is a total of 1918 active links in the dataset. The 289 awesome/recommended links in the dataset are listed below. Feel free to add additional links to the dataset.</p><div>Data Science Specialization in Coursera</div><p>Use R to clean, analyze, and visualize data.</p><div>Learning Statistics with R</div><p>Textbook by Danielle Navarro. Back in the grimdark pre-Snapchat era of humanity (i.e. early 2011), I started teaching an introductory statistics class for psychology students offered at the University of Adelaide, using the R statistical package as the primary tool. I wrote my own lecture notes for the class, which have now expanded to the point of effectively being a book. The book is freely available, and as of version 0.6 it is released under a creative commons licence (CC BY-SA 4.0)</p><div>Data Wrangling and Visualization with R</div><p>This is the website for the Data Wrangling and Visualization with R part of Introduction to Data Science.

The website for the Statistics and Prediction Algorithms Through Case Studies part is here.</p><div>Statistics and Prediction Algorithms Through Case Studies</div><p>This book started out as part of the class notes used in the HarvardX Data Science Series.</p><div>An R companion to Statistics: data analysis and modelling</div><p>A book about how to use R related to the book Statistics: Data analysis and modelling.</p><div>Likelihood Theory</div><p>Likelihood Theory ( by Patrick Breheny)</p><div>Bayesian Regression: Theory & Practice</div><p>This site provides material for an intermediate level course on Bayesian linear regression modeling. The course presupposes some prior exposure to statistics and some acquaintance with R.</p><div>Statistical rethinking with brms, ggplot2, and the tidyverse</div><p>A Solomon Kurz's notes about Statistical rethinking</p><div>Bayes Rules! An Introduction to Applied Bayesian Modeling</div><p>Bayesian statistics?! Once an obscure term outside specialized industry and research circles, Bayesian methods are enjoying a renaissance. The title of this book speaks to what all the fuss is about: Bayes Rules! Bayesian methods provide a powerful alternative to the frequentist methods that are ingrained in the standard statistics curriculum. Though frequentist and Bayesian methods share a common goal – learning from data – the Bayesian approach to this goal is gaining popularity for many reasons: (1) Bayesian methods allow us to interpret new data in light of prior information, formally weaving both into a set of updated information; (2) relative to the confidence intervals and p-values utilized in frequentist analyses, Bayesian results are easier to interpret; (3) Bayesian methods can shine in settings where frequentist “likelihood” methods break down; and (4) the computational tools required for applying Bayesian techniques are increasingly accessible. Unfortunately, the popularity of Bayesian statistics has outpaced the curricular resources needed to support it. To this end, the primary goal of Bayes Rules! is to make modern Bayesian thinking, modeling, and computing accessible to a broader audience.</p><div>An Introduction to Bayesian Data Analysis for Cognitive Science</div><p>This book is intended to be a relatively gentle introduction to carrying out Bayesian data analysis and cognitive modeling using the probabilistic programming language Stan (Carpenter et al. 2017), and the front-end to Stan called brms (Bürkner 2024). Our target audience is cognitive scientists (e.g., linguists, psychologists, and computer scientists) who carry out planned behavioral experiments, and who are interested in learning the Bayesian data analysis methodology from the ground up and in a principled manner. Our aim is to make Bayesian statistics a standard part of the data analysis toolkit for experimental linguistics, psycholinguistics, psychology, and related disciplines.</p><div>Rafalab: Statistical Learning: Algorithmic and Nonparametric Approaches</div><p>Statistical Learning: Algorithmic and
Nonparametric Approaches</p><div>Rafalab:Applied Nonparametric and Modern Statistics</div><p>Applied Nonparametric and Modern Statistics
Official title: Advanced Generalized Linear Models IV 140.753-754</p><div>Regression Modeling Strategies</div><p>All standard regression models have assumptions that must be verified for the model to have power to test hypotheses and for it to be able to predict accurately. Of the principal assumptions (linearity, additivity, distributional), this course will emphasize methods for assessing and satisfying the first two. Practical but powerful tools are presented for validating model assumptions and presenting model results. This course provides methods for estimating the shape of the relationship between predictors and response using the widely applicable method of augmenting the design matrix using restricted cubic splines. Even when assumptions are satisfied, overfitting can ruin a model’s predictive ability for future observations. Methods for data reduction will be introduced to deal with the common case where the number of potential predictors is large in comparison with the number of observations. Methods of model validation (bootstrap and cross-validation) will be covered, as will auxiliary topics such as modeling interaction surfaces, efficiently utilizing partial covariable data by using multiple imputation, variable selection, overly influential observations, collinearity, and shrinkage, and a brief introduction to the R rms package for handling these problems. The methods covered will apply to almost any regression model, including ordinary least squares, longitudinal models, logistic regression models, ordinal regression, quantile regression, longitudinal data analysis, and survival models. Statistical models will be contrasted with machine learning so that the student can make an informed choice of predictive tools.</p><div>Overview | Learning Statistical Models Through Simulation in R</div><p>Textbook on statistical models for social scientists.</p><div>Summary | Mixed Models with R</div><p>This is an introduction to using mixed models in R. It covers the most common techniques employed, with demonstration primarily via the lme4 package. Discussion includes extensions into generalized mixed models, Bayesian approaches, and realms beyond.</p><div>Free Code Camp</div><p>Community for practice programming skills</p><div>Notes of "Data Structures and Algorithms"</div><p>Notes of "Data Structures and Algorithms" (CMPSC 465) at Penn State</p><div>Big Book of R</div><p>R sources collections</p><div>Mastering Software Development in R</div><p>The book covers R software development for building data science tools. As the field of data science evolves, it has become clear that software development skills are essential for producing useful data science results and products. You will obtain rigorous training in the R language, including the skills for handling complex data, building R packages and developing custom data visualizations. You will learn modern software development practices to build tools that are highly reusable, modular, and suitable for use in a team-based environment or a community of developers.</p><div>R Packages (2e)</div><p>Learn how to create a package, the fundamental unit of shareable, reusable, and reproducible R code.</p><div>R for Data Science</div><p>R4DS teaches you how to do data science with R: You’ll learn how to get your data into R, get it into the most useful structure, transform it, visualise it and model it. In this book, you will find a practicum of skills for data science. Just as a chemist learns how to clean test tubes and stock a lab, you’ll learn how to clean data and draw plots—and many other things besides. These are the skills that allow data science to happen, and here you will find the best practices for doing each of these things with R. You’ll learn how to use the grammar of graphics, literate programming, and reproducible research to save time. You’ll also learn how to manage cognitive resources to facilitate discoveries when wrangling, visualising, and exploring data.</p><div>fasteR: Fast Lane to Learning R!</div><p>fasteR: Fast Lane to Learning R!</p><div>R in action</div><p>Good tutorial for R beginner</p><div>Deep R Programming</div><p>Deep R Programming is a comprehensive and in-depth introductory course on one of the most popular languages for data science. It equips ambitious students, professionals, and researchers with the knowledge and skills to become independent users of this potent environment so that they can tackle any problem related to data wrangling and analytics, numerical computing, statistics, and machine learning. This textbook is a non-profit project. Its online and PDF versions are freely available at https://deepr.gagolewski.com/.</p><div>cxli233/Online_R_learning: Online R learning for applied statistics</div><p>Online R learning for applied statistics . Contribute to cxli233/Online_R_learning development by creating an account on GitHub.</p><div>Genomic Data Visualization</div><p>Course Website for EN.580.428 Genomic Data Visualization By JEFworks Lab</p><div>awesome-genome-visualization</div><p>A list of interesting genome visualizers, genome browsers, or genome-browser-like implementations</p><div>ggplot2: Elegant Graphics for Data Analysis (3e)</div><p>This is the on-line version of work-in-progress 3rd edition of “ggplot2: elegant graphics for data analysis” published by Springer. You can learn what’s changed from the 2nd edition in the Preface.</p><div>Datavis.ca</div><p>Friendly's website</p><div>Psyc 6135: Psychology of Data Visualization</div><p>Information visualization is the pictorial representation of data.

Successful visualizations capitalize on our capacity to recognize and understand patterns presented in information displays.
Conversely, they require that writers of scientific papers, software designers and other providers of visual displays understand what works and what does not work to convey their message.
This course will examine a variety of issues related to data visualization from a largely psychological perspective, but will also touch upon other related communities of research and practice related to this topic:

history of data visualization,
computer science and statistical software,
visual design,
human factors.
We will consider visualization methods for a wide range of types of data from the points of view of both the viewer and designer/producer of graphic displays.</p><div>Milestones in the history of thematic cartography, statistical graphics, and data visualization.</div><p>The graphic portrayal of quantitative information has deep roots. These roots reach into histories of thematic cartography, statistical graphics, and data visualization, which are intertwined with each other. They also connect with the rise of statistical thinking up through the 19th century, and developments in technology into the 20th century. From above ground, we can see the current fruit; we must look below to see its pedigree and germination. There certainly have been many new things in the world of visualization; but unless you know its history, everything might seem novel.</p><div>Friends Don't Let Friends Make Bad Graphs</div><p>Friends don't let friends make certain types of data visualization - What are they and why are they bad.

Author: Chenxin Li, Ph.D., Assistant Research Scientist at Department of Crop & Soil Sciences and Center for Applied Genetic Technologies, University of Georgia.</p><div>From Data to Viz</div><p>From Data to Viz leads you to the most appropriate graph for your data. It links to the code to build it and lists common caveats you should avoid.</p><div>Fundamentals of Data Visualization</div><p>Data Visualization Principle</p><div>Vim Notes</div><p>Vim notes by Dave Tang</p><div>How linux works</div><p>Introduction to linux</p><div>Python Distilled</div><p>The richness of modern Python challenges developers at all levels.</p><div>Automate the Boring Stuff with Python</div><div>Tjian + Darzacq Group</div><p>Tij is a professor of biochemistry and molecular biology at the University of California, Berkeley. Trained as a biochemist, he has made major contributions to the understanding of how genes work during three decades at Berkeley. He was named an HHMI investigator in 1987 and served as president of the Howard Hughes Medical Institute from 2009 until 2016.</p><div>Mirny Lab</div><p>The Mirny Lab is a dynamic research group at MIT dedicated to unraveling the mysteries of the 3D genome. We combine cutting-edge experimental techniques, computational modeling, and theoretical insights to understand how the intricate folding of DNA influences gene expression, cell function, and human health.</p><div>Lu Lab</div><p>Lu Lab in Tsinghua University, devoting to developing bioinformatics technologies, and practicing evidence-based precision medicine for diseases like cancer and immune-mediated diseases. We utilize AI technologies and noncoding RNA (ncRNA) centered multi-omics data, to understand how genetic information is encoded in the structured DNA and RNA sequences, and how they interact and regulate each other in a biological system. Ultimately, this will help us understand and cure human diseases, know and improve ourselves.</p><div>a'ayan Laboratory</div><p>The Ma’ayan Laboratory applies machine learning and other statistical mining techniques to study how intracellular regulatory systems function as networks to control cellular processes such as differentiation, dedifferentiation, apoptosis and proliferation.</p><div>Regulatory Genomics Lab</div><p>Regulatory Genomics Lab in Westlake lead by Kai Zhang</p><div>JEFworks Lab</div><p>Bioinformatics research lab in the Center for Computational Biology and the Department of Biomedical Engineering at Johns Hopkins University</p><div>Bioinfo Articles</div><p>Mostly blogs</p><div>Bioinfo Social Media</div><p>Bioinfo twitter and blusky list</p><div>BlueSky Science-Related Feeds</div><div>List : Transcription & Chromatin II</div><div>List: Transcription & Chromatin</div><div>Scientific Journals</div><div>Bioinformatics+Genomics starter pack</div><p>Accounts and feeds to follow in bioinformatics and genomics. High signal, low politics. Prioritizing accounts with interesting original posts (not those that mostly repost). If I've forgotten you, *please* let me know and I'll add you!</p><div>List: single-cell computational methods</div><div>List: Genomics, Evolution, and More. pt1</div><div>List: Algorithmic Genomics</div><p>Peeps actively working on algorithms and data structures for omics data.

NOTE: This list started off-hand, based on ppl I see actively on this platform, and is certainly highly incomplete. If you think you belong here, let me know!</p><div>List: Genomics, Evolution, and More. pt2</div><div>Genomics Technologies</div><div>List: Cancer research</div><p>A starter pack of curious cancer researchers - from immunology to genomics to cell biology and so much more</p><div>List: Metagenomics, microbial genomics</div><p>Genomicists who study microbial genomes and communities</p><div>Data Story</div><p>A podcast on data visualization with Enrico Bertini and Moritz Stefaner, http://datastori.es/ . Interviews with over 100 graphic designers & developers.</p><div>Night Science</div><p>Where do ideas come from? In each episode, scientists Itai Yanai and Martin Lercher explore science's creative side with a leading colleague. New episodes come out every second Monday.</p><div>MirnyLab</div><p>Videos for publications of Mirny Lab @ MIT 
http://mirnylab.mit.edu</p><div>StatQuest with Josh Starmer</div><p>Learn statistics in a easy way. BAM!!!!</p><div>Xiaole Shirley Liu</div><p>Xiaole Shirley Liu's YouTube</p><div>3Blue1Brown</div><p>3Blue1Brown. Math Youtube</p><div>Manolis Kellis</div><p>Manolis Kellis' YouTube</p><div>Towards Data Science</div><p>Towards Data Science started in 2016 as a community-powered hub where data science and machine learning practitioners can share their knowledge and ideas with their peers — regardless of their location or background. This fundamental goal remains the same today.</p><div>Biostar forum</div><p>Bioinfomatics forum</p><div>Visualising Data</div><p>ANDY KIRK's blog. Visualisingdata.com was originally launched in 2010 originally to serve as a blog to help continue the momentum of my learning from studying the subject via a Masters degree. I continue to publish articles and share announcements that track developments in my professional experiences as well as developments in the data visualisation field at large.</p><div>Junk Chart</div><p>Kaiser Fung. Business analytics and data visualization expert. Author and Speaker.</p><div>Flowing data</div><p>Data visualization blog</p><div>R some blog</div><p>Roman Pahl's blog</p><div>R bloggers</div><p>R news and tutorials contributed by hundreds of R bloggers</p><div>Hilary Parker</div><p>Hello, I’m Hilary Parker! I’m a Data Scientist, previously of Stitch Fix, Etsy, and the 2020 Biden for President Campaign. I&#039;m passionate about the intersection of data science and product, from deeply understanding users to designing new experiences that depend on innovative data pipelines and client interactions. My work from Stitch Fix was featured in&hellip;</p><div>Dave Tang's blog</div><p>Computational biology and genomics</p><div>Dave Tang's blog</div><p>Computational biology and genomics</p><div>Karl Broman's blog</div><p>Karl Broman's blog</p><div>Bits of DNA</div><p>Reviews and commentary on computational biology by Lior Pachter</p><div>Bioinformatics and other bits</div><p>Damien Farrell's blog</p><div>CHATOMICS</div><p>Ming Tang's blogs</p><div>Gilbert Han</div><p>Oh! This is my blog.</p><div>Paired Ends | Substack</div><p>Stephen Turner's blog</p><div>The Book of Why: The New Science of Cause and Effect</div><p>"Correlation is not causation." This mantra, chanted by scientists for more than a century, has led to a virtual prohibition on causal talk. Today, that taboo is dead. The causal revolution, instigated by Judea Pearl and his colleagues, has cut through a century of confusion and established causality -- the study of cause and effect -- on a firm scientific basis. His work explains how we can know easy things, like whether it was rain or a sprinkler that made a sidewalk wet; and how to answer hard questions, like whether a drug cured an illness. Pearl's work enables us to know not just whether one thing causes another: it lets us explore the world that is and the worlds that could have been. It shows us the essence of human thought and key to artificial intelligence. Anyone who wants to understand either needs The Book of Why.</p><div>Lewin's GENES XII</div><p>Long considered the quintessential molecular biology textbook, for decades Lewin's GENES has provided the most modern presentation to this transformative and dynamic science. Now in its twelfth edition, this classic text continues to lead with new information and cutting-edge developments, covering gene structure, sequencing, organization, and expression. Leading scientists provide revisions and updates in their respective areas of study offering readers current research and relevant information on the rapidly changing subjects in molecular biology. No other text offers a broader understanding of this exciting and vital science or does so with higher quality art and illustrations. Lewin's GENES XII continues to be the clear choice for molecular biology and genetics.</p><div>An Owner's Guide to the Human Genome: an introduction to human population genetics, variation and disease</div><p>In this book I describe the forces that govern genetic variation including mutation, drift, recombination and selection, as well as what genetics teaches us about human history, and the role of genetic variation in human phenotypes and diseases. When complete, the book will combine the three pillars of human population genetics - population genetics, population history, and trait genetics - under a single umbrella, with a focus on examples and applications in human genetics. Moreover, each section emphasizes the essential interplay between theory, statistical methods, and biological applications, with a focus on building intuition while avoiding heavy technical detail where possible.</p><div>Human Genome Variation</div><p>This is the course homepage and digital textbook for Human Genome Variation with Computational Lab</p></div><!-- nav.seo-end -->
  <script src="runtime.d9cda80d51f23b51.js" type="module"></script><script src="polyfills.ba9b77532fd27fc9.js" type="module"></script><script src="main.0c9afba02a0b1d55.js" type="module"></script></body>
</html>
