<!DOCTYPE html>
<!--
                                                                          
                                 .--,-``-.                              
                                /   /     '.                            
                               / ../        ;                           
      ,---,                    \ ``\  .`-    '                   ,---,  
  ,-+-. /  |               .---.\___\/   \   :               ,-+-. /  | 
 ,--.'|'   |  ,--.--.    /.  ./|     \   :   |      ,---.   ,--.'|'   | 
|   |  ,"' | /       \ .-' . ' |     /  /   /      /     \ |   |  ,"' | 
|   | /  | |.--.  .-. /___/ \: |     \  \   \     /    / ' |   | /  | | 
|   | |  | | \__\/: . .   \  ' . ___ /   :   |   .    ' /  |   | |  | | 
|   | |  |/  ," .--.; |\   \   '/   /\   /   :   '   ; :__ |   | |  |/  
|   | |--'  /  /  ,.  | \   \  / ,,/  ',-    .___'   | '.'||   | |--'   
|   |/     ;  :   .'   \ \   \ \ ''\        ;/  .\   :    :|   |/       
'---'      |  ,     .-./  '---" \   \     .' \  ; \   \  / '---'        
            `--`---'             `--`-,,-'    `--" `----'               
                      
                     2018/4/26   www.nav3.cn

        ,+++++++++++++++++++++++++++++++++++++++++++++++++.
        ( We want you! https://github.com/xjh22222228/nav )
        `-,+++++++++++++++++++++++++++++++++++++++++++++++'
-->
<html data-critters-container>
  <head>
    <meta charset="utf-8">
    <base href="./">
    <meta name="author" content="https://github.com/xjh22222228">
    <meta http-equiv="X-UA-Compatible" content="IE=edge, chrome=1">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
    <meta name="renderer" content="webkit">
    <!-- nav.config-start --><!-- https://github.com/xjh22222228/nav -->
  <title>Bioinfo Explore</title>
  <meta property="og:title" content="Bioinfo Explore">
  <meta property="og:description" content="Curated Bioinformatics Resources">
  <meta property="og:type" content="website">
  <meta name="description" content="Curated Bioinformatics Resources">
  <meta name="keywords" content="Bioinfomatics, Navigation, Programming" id="xjh_2">
  <link rel="icon" href="https://gcore.jsdelivr.net/gh/xjh22222228/public@gh-pages/nav/logo.svg">
  <link rel="apple-touch-icon" href="https://gcore.jsdelivr.net/gh/xjh22222228/public@gh-pages/nav/logo.svg">
  <link rel="prefetch" href="//unpkg.com/ng-zorro-antd@18.1.1/ng-zorro-antd.dark.min.css"><!-- nav.config-end -->
    <link rel="stylesheet" href="assets/fonts/iconfont.css" media="print" onload="this.media='all'"><noscript><link rel="stylesheet" href="assets/fonts/iconfont.css"></noscript>
    <!-- nav.headerContent-start -->
    <!-- nav.headerContent-end -->
  <style>*,:before,:after{--tw-border-spacing-x:0;--tw-border-spacing-y:0;--tw-translate-x:0;--tw-translate-y:0;--tw-rotate:0;--tw-skew-x:0;--tw-skew-y:0;--tw-scale-x:1;--tw-scale-y:1;--tw-pan-x: ;--tw-pan-y: ;--tw-pinch-zoom: ;--tw-scroll-snap-strictness:proximity;--tw-gradient-from-position: ;--tw-gradient-via-position: ;--tw-gradient-to-position: ;--tw-ordinal: ;--tw-slashed-zero: ;--tw-numeric-figure: ;--tw-numeric-spacing: ;--tw-numeric-fraction: ;--tw-ring-inset: ;--tw-ring-offset-width:0px;--tw-ring-offset-color:#fff;--tw-ring-color:rgb(59 130 246 / .5);--tw-ring-offset-shadow:0 0 #0000;--tw-ring-shadow:0 0 #0000;--tw-shadow:0 0 #0000;--tw-shadow-colored:0 0 #0000;--tw-blur: ;--tw-brightness: ;--tw-contrast: ;--tw-grayscale: ;--tw-hue-rotate: ;--tw-invert: ;--tw-saturate: ;--tw-sepia: ;--tw-drop-shadow: ;--tw-backdrop-blur: ;--tw-backdrop-brightness: ;--tw-backdrop-contrast: ;--tw-backdrop-grayscale: ;--tw-backdrop-hue-rotate: ;--tw-backdrop-invert: ;--tw-backdrop-opacity: ;--tw-backdrop-saturate: ;--tw-backdrop-sepia: ;--tw-contain-size: ;--tw-contain-layout: ;--tw-contain-paint: ;--tw-contain-style: }*,:before,:after{box-sizing:border-box;border-width:0;border-style:solid;border-color:#e5e7eb}:before,:after{--tw-content:""}html{line-height:1.5;-webkit-text-size-adjust:100%;tab-size:4;font-family:ui-sans-serif,system-ui,sans-serif,"Apple Color Emoji","Segoe UI Emoji",Segoe UI Symbol,"Noto Color Emoji";font-feature-settings:normal;font-variation-settings:normal;-webkit-tap-highlight-color:transparent}body{margin:0;line-height:inherit}p{margin:0}html,body{width:100%;height:100%}*,*:before,*:after{box-sizing:border-box}html{font-family:sans-serif;line-height:1.15;-webkit-text-size-adjust:100%;-ms-text-size-adjust:100%;-ms-overflow-style:scrollbar;-webkit-tap-highlight-color:rgba(0,0,0,0)}body{margin:0;color:#000000d9;font-size:14px;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,"Apple Color Emoji","Segoe UI Emoji",Segoe UI Symbol,"Noto Color Emoji";font-variant:tabular-nums;line-height:1.5715;background-color:#fff;font-feature-settings:"tnum"}p{margin-top:0;margin-bottom:1em}html{--antd-wave-shadow-color:#1890ff;--scroll-bar:0}@charset "UTF-8";:root{--componentHeight:155px}html{-ms-text-size-adjust:100%;-webkit-text-size-adjust:100%}body{min-height:100vh;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;-webkit-overflow-scrolling:touch;font-size:14px}</style><link rel="stylesheet" href="styles.341d5df18e498c5e.css" media="print" onload="this.media='all'"><noscript><link rel="stylesheet" href="styles.341d5df18e498c5e.css"></noscript></head>
  <script>
    window.__TITLE__ = document.title
  </script>
  <body class="dark-scrollbar-border" id="x-j-h">
    <app-xiejiahe>
      <!-- nav.loading-start --><style>.l__1{z-index:999;position:fixed;top:0;left:0;right:0;bottom:0;background-color:#fff;display:flex;align-items:center;justify-content:center;flex-direction:column;}.l__2{width:200px;height:200px;background:url("https://gcore.jsdelivr.net/gh/xjh22222228/public@gh-pages/nav/0020f4d5-9f70-4b7f-87d4-61310d1db8d1.svg");background-repeat:repeat;}.l__3{font-size:18px;margin-top:15px;font-weight:bold;color:#000;}</style>
<div class="l__1"><div class="l__2"></div><div class="l__3">......</div></div><!-- nav.loading-end -->
    </app-xiejiahe>

    <!-- nav.seo-start -->
<div data-url="https://github.com/xjh22222228/nav" data-server-time="1731221771698" data-a="x.i.e-jiahe" data-date="2024-11-10 14:56:11" id="META-NAV" style="z-index:-1;position:fixed;top:-10000vh;left:-10000vh;">
<div>Regulatory Genomics Lab</div><p>Regulatory Genomics Lab in Westlake lead by KaiZhang</p><div>JEFworks Lab</div><p>Bioinformatics research lab in the Center for Computational Biology and the Department of Biomedical Engineering at Johns Hopkins University</p><div>Night Science</div><p>Where do ideas come from? In each episode, scientists Itai Yanai and Martin Lercher explore science's creative side with a leading colleague. New episodes come out every second Monday.</p><div>StatQuest with Josh Starmer</div><p>Learn statistics in a easy way. BAM!!!!</p><div>Xiaole Shirley Liu</div><p>Xiaole Shirley Liu's YouTube</p><div>3Blue1Brown</div><p>3Blue1Brown. Math Youtube</p><div>Manolis Kellis</div><p>Manolis Kellis' YouTube</p><div>Biostar forum</div><p>Bioinfomatics forum</p><div>R some blog</div><p>Roman Pahl's blog</p><div>R bloggers</div><p>R news and tutorials contributed by hundreds of R bloggers</p><div>Hilary Parker</div><p>Hello, I’m Hilary Parker! I’m a Data Scientist, previously of Stitch Fix, Etsy, and the 2020 Biden for President Campaign. I&#039;m passionate about the intersection of data science and product, from deeply understanding users to designing new experiences that depend on innovative data pipelines and client interactions. My work from Stitch Fix was featured in&hellip;</p><div>Bits of DNA</div><p>Reviews and commentary on computational biology by Lior Pachter</p><div>Bioinformatics and other bits</div><p>Damien Farrell's blog</p><div>CHATOMICS</div><p>Ming Tang's blogs</p><div>Gilbert Han</div><p>Oh! This is my blog.</p><div>Paired Ends | Substack</div><p>Stephen Turner's blog</p><div>refine.bio</div><p>refine.bio Compendia packages the data processed by refine.bio pipelines for flexible and broad use by computational biologists and data scientists</p><div>Single-cell Pan-species atlas</div><p>SPEED is the largest pan-species single cell database at present, covering 127 species to date. SPEED also collects various scRNA-seq atlases in evolution, development and disease, providing an ecological and evolutionary perspective for the study of development and disease.</p><div>Single Cell Portal</div><p>Single Cell Portal by BroadInstitute</p><div>CELLxGENE</div><p>scRNA datasets collections</p><div>Zhonglab's 3D genome tutorial</div><p>Zhonglab's 3D genome tutorial</p><div>Ming Tang's ChIP-seq notes</div><p>Ming Tang's ChIP-seq notes</p><div>Hi-C Data Analysis Bootcamp</div><p>Hi-C Data Analysis Bootcamp</p><div>Github: Hi-C data analysis tools and papers</div><p>Hi-C data analysis tools and papers collection</p><div>Orchestrating Hi-C analysis with Bioconductor</div><p>Orchestrating Hi-C analysis with Bioconductor</p><div>Cooltools  documentation</div><p>Cooltools tutorial</p><div>Bioinformatics Algorithms</div><p>Bioinformatics algorithms book</p><div>ROSALIND</div><p>ROSALIND programming website</p><div>Computational Genomics with R</div><p>The aim of this book is to provide the fundamentals for data analysis for genomics. We developed this book based on the computational genomics courses we are giving every year.</p><div>Modern Statistics for Modern Biology</div><p>Modern Statistics for Modern Biology, by Susan Holmes and Wolfgang Huber</p><div>Misc notes</div><p>Various notes collected over the years</p><div>The Biostar Handbook: 2nd Edition</div><p>A guide to bioinformatics data analysis</p><div>GitHub - crazyhottommy/getting-started-with-genomics-tools-and-resources: Unix, R and python tools for genomics and data science</div><p>Unix, R and python tools for genomics and data science - crazyhottommy/getting-started-with-genomics-tools-and-resources</p><div>harvardx</div><p>HarvardX Biomedical Data Science By rafalab</p><div>Introduction to Bioinformatics and Computational Biology</div><p>Introduction to Bioinformatics and Computational Biology By Xiaole Liu Lab</p><div>Orchestrating Single-Cell Analysis with Bioconductor</div><p>Bioconductor's scRNA tutorial</p><div>Single-cell best practices</div><div>Data Wrangling and Visualization with R</div><p>This is the website for the Data Wrangling and Visualization with R part of Introduction to Data Science.

The website for the Statistics and Prediction Algorithms Through Case Studies part is here.</p><div>Statistics and Prediction Algorithms Through Case Studies</div><p>This book started out as part of the class notes used in the HarvardX Data Science Series.</p><div>An R companion to Statistics: data analysis and modelling</div><p>A book about how to use R related to the book Statistics: Data analysis and modelling.</p><div>Overview | Learning Statistical Models Through Simulation in R</div><p>Textbook on statistical models for social scientists.</p><div>Summary | Mixed Models with R</div><p>This is an introduction to using mixed models in R. It covers the most common techniques employed, with demonstration primarily via the lme4 package. Discussion includes extensions into generalized mixed models, Bayesian approaches, and realms beyond.</p><div>Big Book of R</div><p>R sources collections</p><div>Mastering Software Development in R</div><p>The book covers R software development for building data science tools. As the field of data science evolves, it has become clear that software development skills are essential for producing useful data science results and products. You will obtain rigorous training in the R language, including the skills for handling complex data, building R packages and developing custom data visualizations. You will learn modern software development practices to build tools that are highly reusable, modular, and suitable for use in a team-based environment or a community of developers.</p><div>R Packages (2e)</div><p>Learn how to create a package, the fundamental unit of shareable, reusable, and reproducible R code.</p><div>fasteR: Fast Lane to Learning R!</div><p>fasteR: Fast Lane to Learning R!</p><div>R in action</div><p>Good tutorial for R beginner</p><div>Deep R Programming</div><p>Deep R Programming is a comprehensive and in-depth introductory course on one of the most popular languages for data science. It equips ambitious students, professionals, and researchers with the knowledge and skills to become independent users of this potent environment so that they can tackle any problem related to data wrangling and analytics, numerical computing, statistics, and machine learning. This textbook is a non-profit project. Its online and PDF versions are freely available at https://deepr.gagolewski.com/.</p><div>cxli233/Online_R_learning: Online R learning for applied statistics</div><p>Online R learning for applied statistics . Contribute to cxli233/Online_R_learning development by creating an account on GitHub.</p><div>Genomic Data Visualization</div><p>Course Website for EN.580.428 Genomic Data Visualization By JEFworks Lab</p><div>awesome-genome-visualization</div><p>A list of interesting genome visualizers, genome browsers, or genome-browser-like implementations</p><div>Fundamentals of Data Visualization</div><p>Data Visualization Principle</p><div>Vim Notes</div><p>Vim notes by Dave Tang</p><div>How linux works</div><p>Introduction to linux</p><div>Python Distilled</div><p>The richness of modern Python challenges developers at all levels.</p><div>Automate the Boring Stuff with Python</div></div><!-- nav.seo-end -->
  <script src="runtime.d9cda80d51f23b51.js" type="module"></script><script src="polyfills.ba9b77532fd27fc9.js" type="module"></script><script src="main.0d127f506c769bcd.js" type="module"></script></body>
</html>
